Vagrantfile

```diff
@@ -11,14 +12,11 @@ Vagrant.configure("2") do |config|
     ansible.inventory_path = "inventory_local.yml"
     ansible.limit = "all"
     ansible.become = true
-    ansible.install_mode = "pip_args_only"
-    ansible.pip_args = "-r /vagrant/requirements.txt"
-    ansible.pip_install_cmd = "sudo apt-get install -y python3-pip python-is-python3 haveged && sudo ln -s -f /usr/bin/pip3 /usr/bin/pip"
     ansible.galaxy_role_file = "requirements.yml"
     ansible.extra_vars = {
       nexus_user: ENV['NEXUS_USERNAME'],
       nexus_password: ENV['NEXUS_PASSWORD'],
       autogen_unsecure_secrets: true
     }
```

Comment:
ansible_local provision should be unnecessary now - remove it

Reply:
> ansible_local provision should be unnecessary now - remove it

What about ansible.extra_vars? nexus credentials?